paket.dependencies

```diff
@@ -10,6 +10,7 @@ nuget FSharpLint.Core prerelease
 nuget VSSDK.IDE.Deploy.14
 nuget FSharp.Management
 nuget FsPickler
+nuget FSharp.Interop.Dynamic
```

Comment:
Adding this dependency for a single use seems a bit overkill. We have used dynamic in the code base using this https://github.com/fsprojects/VisualFSharpPowerTools/blob/e3b2531d83e008360c1fd6bcdb0bcfb25c9aaa31/src/FSharpVSPowerTools.Logic/ProjectSystem.fs#L15. Could it eliminate the need of an extra dependency?